src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -36,6 +36,8 @@ namespace System.Globalization
 #endif
     sealed partial class IdnMapping
     {
+        private const int StackallocThreshold = 512;
+
```

Comment:
Is this used by the Windows implementation as well?  If not, I think it should be moved back to the Unix-specific file.